src/components/signature/Signature.ts

```diff
@@ -302,8 +309,32 @@ export class Signature {
     }
   }
 
+  private registerTouchmove(e: TouchEvent) {
+    this.registerTouchEvent(e, 'mousemove')
+  }
+  
+  private registerTouchstart(e: TouchEvent) {
+    this.registerTouchEvent(e, 'mousedown')
+  }
+
+  private registerTouchend() {
+    const me = new MouseEvent('mouseup', {});
+    this.canvas.dispatchEvent(me);
+  }
+
+  private registerTouchEvent(e: TouchEvent, eventName: string) {
+    const touch = e.touches[0];
+    const me = new MouseEvent(eventName, {
+      clientX: touch.clientX,
+      clientY: touch.clientY
+    });
+    this.canvas.dispatchEvent(me);
+  }
+
   private _dispose() {
     this.mask.remove()
     this.container.remove()
+    document.querySelector<HTMLElement>('html')!.style.overflow = ''
```

Comment:
You should restore the previously set state instead of directly clearing it

Reply:
Do you think it's better to create a css class and add/remove as needed?